pkg/k8s/apis/cilium.io/utils/repository.go

```diff
@@ -0,0 +1,141 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package utils
+
+import (
+	slim_metav1 "github.com/cilium/cilium/pkg/k8s/slim/k8s/apis/meta/v1"
+	"github.com/cilium/cilium/pkg/policy/api"
+	"github.com/cilium/cilium/pkg/policy/types"
+)
+
+// RulesToPolicyEntries converts a slice of api.Rules to PolicyEntries
```

Comment:
nit: the comment tells me exactly as much as the type signature of the function - would be more interesting to have a comment here which explains (briefly) _why_ we want to have this in the first place. Something like "RulestoPolicyEntries converts the external rule type to the internal policy entry representation" which already gives a bit more context